src/sentry/hybridcloud/migrations/0005_add_api_and_org_token_replicas.py

```diff
@@ -0,0 +1,139 @@
+                            "team:read",
+                            "team:write",
+                            "team:admin",
+                            "event:read",
+                            "event:write",
+                            "event:admin",
+                            "org:read",
+                            "org:write",
+                            "org:admin",
+                            "member:read",
+                            "member:write",
+                            "member:admin",
+                        ],
+                        default=None,
+                    ),
+                ),
+                ("scope_list", sentry.db.models.fields.array.ArrayField(null=True)),
+                (
+                    "application_id",
+                    sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                        "sentry.ApiApplication", db_index=True, null=True, on_delete="CASCADE"
+                    ),
+                ),
+                (
+                    "organization_id",
+                    sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                        "sentry.Organization", db_index=True, null=True, on_delete="SET_NULL"
+                    ),
+                ),
+                ("application_is_active", models.BooleanField(default=False)),
+                (
+                    "user_id",
+                    sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                        "sentry.User", db_index=True, on_delete="CASCADE"
+                    ),
+                ),
+                (
+                    "apitoken_id",
+                    sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                        "sentry.ApiToken", db_index=True, on_delete="CASCADE"
+                    ),
+                ),
+                ("token", models.CharField(max_length=64)),
+                ("expires_at", models.DateTimeField(null=True)),
+                ("allowed_origins", models.TextField(blank=True, null=True)),
+                ("date_added", models.DateTimeField(default=django.utils.timezone.now)),
+            ],
+            options={
+                "db_table": "hybridcloud_apitokenreplica",
+            },
+        ),
+        migrations.CreateModel(
+            name="OrgAuthTokenReplica",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                (
+                    "orgauthtoken_id",
+                    sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                        "sentry.OrgAuthToken", db_index=True, on_delete="CASCADE"
+                    ),
+                ),
+                ("token_hashed", models.TextField()),
```

Comment:
Should this be unique=True to prevent duplicates even if replication goes sideways?